src/sonarwhal-theme/scripts/link-filter.js

```diff
@@ -22,6 +24,10 @@ hexo.extend.filter.register('before_post_render', (data) => {
             matchHtmlUrl = matchHtmlUrl.replace(/\((.*?)\)/g, '(../$1)');
         }
 
-        data.content = data.content.replace(matchMdUrl, matchHtmlUrl);
+        // ../../../../rule-axe/README/ => rule-axe
+        const matchRuleUrl = matchHtmlUrl.match(ruleUrlRegex);
+        const newUrl = isRulePage && matchRuleUrl ? `(${matchRuleUrl.pop()})` : matchHtmlUrl;
```

Comment:
> `(${matchRuleUrl.pop()})` =>  `(${matchRuleUrl.pop()}/)`

@qzhou1607 Added a tailing `/` because of https://github.com/sonarwhal/sonarwhal.com/issues/393.